concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,160 @@
+# About
+
+## 1. Regular Expressions in JavaScript
```

Comment:
We don't user numbers in headers anywhere else so it would be better to remove them. That also makes it easier for contributors to add/change things later.